solutions/security/explore/users-page.md

```diff
@@ -65,7 +34,7 @@ Beneath the KPI charts are data tables, which are useful for viewing and investi
 * **All users**: A chronological list of unique user names, when they were last active, and the associated domains.
 * **Authentications**: A chronological list of user authentication events and associated details, such as the number of successes and failures, and the host name of the last successful destination.
 * **Anomalies**: Unusual activity discovered by machine learning jobs that contain user data.
-* **User risk**: The latest recorded user risk score for each user, and its user risk classification. This feature requires a [Platinum subscription](https://www.elastic.co/pricing) or higher and must be enabled to display the data. Click **Enable** on the **User risk** tab to get started. To learn more, refer to our [entity risk scoring documentation](/solutions/security/advanced-entity-analytics/entity-risk-scoring.md).
+* **User risk**: The latest recorded user risk score for each user, and its user risk classification. In {{stack}} 9.0.0+, this feature requires a [Platinum subscription](https://www.elastic.co/pricing) or higher. In serverless, this feature requires the Security Analytics Complete [project feature](/deploy-manage/deploy/elastic-cloud/project-settings.md). Click **Enable** on the **User risk** tab to get started. To learn more, refer to our [entity risk scoring documentation](/solutions/security/advanced-entity-analytics/entity-risk-scoring.md).
```

Comment:
Nice use of the [prose versioning guidelines](https://docs-v3-preview.elastic.dev/elastic/docs-builder/tree/main/versions/content-patterns#prose)!